.buildkite/core.rayci.yml

```diff
@@ -339,5 +339,6 @@ steps:
       - bazel run //ci/ray_ci:test_in_docker -- //python/ray/tests/... //python/ray/dag/... core
         --parallelism-per-worker 2 --gpus 2
         --build-name coregpubuild
-        --only-tags multi_gpu
+        --only-tags multi_gpu || true
+      - sleep 1000000
```

Comment:
w00t i think you might forget to remove this on CI so it was running for 8 hours